Gemfile

```diff
@@ -63,6 +63,15 @@ group :development, :test do
   #       thin
   # Hence as dependency with thin is needed we have to install it separately
   gem 'thin', '~> 1.8', '>= 1.8.2'
+
+  # Money-rails: Integrates the Money gem with Rails
+  # Provides a comprehensive solution for handling money and currency conversions
+  gem 'money-rails'
+
+  # OXR: Ruby wrapper for the Open Exchange Rates API
+  # Allows fetching real-time and historical exchange rates
+  gem 'oxr'
```

Comment:
Can you share which documentation you have used for this? 
I think we can use this gem https://github.com/spk/money-open-exchange-rates
It has better rate-fetching code with money-rails gem.
You have a detailed view here: https://www.youtube.com/watch?v=FzUZRdn7Qi4&list=PLoDt3UyLUtciCuWiX4ZV-oVCei-8xXH4v&index=7

Reply:
I did search using AI that has provided this gem, have modified to the suggested gem